lib/PRC/Controller/Auth.pm

```diff
@@ -91,7 +92,9 @@ sub callback :Path('/callback') :Args(0) {
     # If that didn't work, kick out
     $c->forward('login_error') unless $user;
   }
-
+  if($user->github_login eq PRC::Secrets->init_admin) {
```

Comment:
Let's not do the init_admin thing. I'd rather manually assign admin roles for now.